docs/src/docs/asciidoc/getting-started.adoc

```diff
@@ -163,25 +167,29 @@ the configuration are described in the following listings:
 	}
 
 	asciidoctor { <6>
-		inputs.dir snippetsDir <7>
-		dependsOn test <8>
+		dependsOn test <7>
+		configurations 'asciidoctorExt'
+		baseDirFollowsSourceDir() <8>
```

Comment:
AFAIK, this isn't always required so I'd prefer not to document it here.

Reply:
It is not required but not using it was bring some trouble when I moved my project from maven. Without this Asciidoctor will consider the project root as base dir for any include.

e.g.
Previously I had this: `:pkgsrc: ../../main/java/com/brneto/rest/reactive`
And I'd need to change for this `:pkgsrc: src/main/java/com/brneto/rest/reactive`

Reply:
As it's unrelated to the change at hand (updating to use Gradle 7-compatible configurations) it should be considered separately.

Reply:
So you want I remove it and let the default behaviour?

Reply:
Yes please.

Reply:
Do you think would be good to mention the location of the base dir, then?

Reply:
It certainly shouldn't be done as part of these changes as it's unrelated. We can consider adding something separately, although I'm not sure it belong in REST Docs' documentation as it's a general Asciidoctor and Gradle concern.

Reply:
Done!